### PR TITLE
docs: 2.18 changelog

### PR DESCRIPTION
**Pull request by mikehardy**

### CODE REVIEW

File: changelog.asc [hunk 1]
```diff
@@ -2,6 +2,45 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.18.0 (20240512)
+* Storage Migration blocked 2.17 for a year, we are back on a more rapid release cadence now. Enjoy!
```
Comment (david-allison) [inline changelog.asc:6]: Storage migration blocked 2.16, Anki backend blocked 2.17

File: changelog.asc [hunk 2]
```diff
@@ -2,6 +2,45 @@
...
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** New Previewer
```
Comment (david-allison) [inline changelog.asc:9]:
```suggestion
** Newly designed previewer
```

File: changelog.asc [hunk 3]
```diff
@@ -2,6 +2,45 @@
...
+** Improved Media Syncing
```
Comment (david-allison) [inline changelog.asc:10]:
```suggestion
** Improved media syncing user experience
```

File: changelog.asc [hunk 4]
```diff
@@ -2,6 +2,45 @@
...
+** Delete Account up to date with new Google requirements
+** Set Due Date
```
Comment (david-allison) [inline changelog.asc:12]:
```suggestion
** Additional 'Set Due Date' options: select from date range & optionally update interval
```

File: changelog.asc [hunk 5]
```diff
@@ -2,6 +2,45 @@
...
+** Forget Cards
```
Comment (david-allison) [inline changelog.asc:13]:
```suggestion
** Forget Cards: support restoring queue positions and resetting lapse counts
```

File: changelog.asc [hunk 6]
```diff
@@ -2,6 +2,45 @@
...
+** Toggle Bury
```
Comment (david-allison) [inline changelog.asc:14]:
```suggestion
** Toggle 'bury' status of multiple cards
```

File: changelog.asc [hunk 7]
```diff
@@ -2,6 +2,45 @@
...
+** Show progress when optimizing all presets
+** Improved 'Check Pronunciation'
```
Comment (david-allison) [inline changelog.asc:16]:
```suggestion
** 'Check Pronunciation': streamlined design
```

File: changelog.asc [hunk 8]
```diff
@@ -2,6 +2,45 @@
...
+** Add keyboard shortcuts to flag and mark in previewer
+** Undo stroke in drawing editor
+** Heirarchical deck selection in Note Editor
+*** Hint: Propose a leading zero to ensure ordering of decks in Deck Picker [01,, 02, 10]
```
Comment (david-allison) [inline changelog.asc:20]:
```suggestion
*** Deck Creation: Improved hints for numerical orderings in Deck Picker
```

File: changelog.asc [hunk 9]
```diff
@@ -2,6 +2,45 @@
...
+*** Inform a user of how to lexicographically order decks
```
Comment (david-allison) [inline changelog.asc:21]:
```suggestion
```

File: changelog.asc [hunk 10]
```diff
@@ -2,6 +2,45 @@
...
+** Hide Toolbar on Scroll
```
Comment (david-allison) [inline changelog.asc:22]:
```suggestion
** Statistics: Hide toolbar on scroll
```

File: changelog.asc [hunk 11]
```diff
@@ -2,6 +2,45 @@
...
+** Search Bar in 'Manage Note Types'
```
Comment (david-allison) [inline changelog.asc:23]:
```suggestion
** Manage Note Types: Add search bar
```

File: changelog.asc [hunk 12]
```diff
@@ -2,6 +2,45 @@
...
+** Match the color of status bar with Statistic toolbar
```
Comment (david-allison) [inline changelog.asc:24]:
```suggestion
** Match the color of status bar with Statistics toolbar
```

File: changelog.asc [hunk 13]
```diff
@@ -2,6 +2,45 @@
...
+* Fixes
+** Video Playback
```
Comment (david-allison) [inline changelog.asc:26]:
```suggestion
** Automatic Video Playback
```

File: changelog.asc (hunk 1 above)
Comment (mikehardy) [reply changelog.asc:6]: urf, exactly right
I'm still PTSD on storage migration I guess

Comment (mikehardy) [inline changelog.asc:6]:
```suggestion
* Switching to more direct use of Anki blocked 2.17 for a year, but with 2.18 we are back on a more rapid release cadence. Enjoy!
```

File: changelog.asc [hunk 14]
```diff
@@ -2,6 +2,44 @@
...
+** Improved media syncing user experience
+** Delete Account up to date with new Google requirements
+** Additional 'Set Due Date' options: select from date range & optionally update interval
+** Forget Cards: support restoring queue positions and resetting lapse counts
+** Toggle 'bury' status of multiple cards
+** Show progress when optimizing all presets
+** 'Check Pronunciation': streamlined design
+** Add keyboard shortcuts to flag and mark in previewer
+** Undo stroke in drawing editor
+** Heirarchical deck selection in Note Editor
```
Comment (david-allison) [inline changelog.asc:19]: Hierarchical

**Pull request merged**